src/elements/Typography.tsx

```diff
@@ -68,6 +68,7 @@ export interface TextProps
   fontFamily?: string
   fontSize: number
   lineHeight: number
+  style: CSSProperties
```

Comment:
This is a required prop, is that intentional? The description of the PR makes it sound optional.

Reply:
Ah, good catch -- fixed 👍